otherlibs/stdune/src/appendable_list.mli

```diff
@@ -9,3 +9,6 @@ val ( @ ) : 'a t -> 'a t -> 'a t
 val cons : 'a -> 'a t -> 'a t
 val concat : 'a t list -> 'a t
 val to_list : 'a t -> 'a list
+val to_list_rev : 'a t -> 'a list
+val of_list : 'a list -> 'a t
```

Comment:
I like the addition of `to_list_rev`, since it makes the function tail-recursive without paying the extra cost of reversing the list at the end. Do we even need `to_list` now? I realize its usefulness depends on the context, but it seems kinda nice to delete it altogether so that we can't accidentally use it instead of `to_list_rev` without thinking.

Reply:
No, we don't use it. I just added it for completeness. I'll get rid of it.